### ISSUE TITLE
Typo in the Github Checkout Action

### ISSUE THREAD

**Issue by sjperkins** [ska-sa/tricolour#72]
* Tricolour version: master
* Python version: 3.6
* Operating System: 18.04

### Description

From here https://github.com/ska-sa/tricolour/runs/1288246393

```
Prepare workflow directory
Prepare all required actions
Getting action download info
Failed to resolve action download info. Error: Unable to resolve action `actions/chcekout@v2`, repository not found
Retrying in 15.406 seconds
Failed to resolve action download info. Error: Unable to resolve action `actions/chcekout@v2`, repository not found
Retrying in 24.486 seconds
Error: Unable to resolve action `actions/chcekout@v2`, repository not found
```
@smasoka Can you please take a look?

**Comment by smasoka** [comment]
New PR to fix the spelling error.



* SAKHILE MASOKADATA SCIENTISTCell: +27 (0)72 045
2240Email: sakhile.harvey@gmail.com <sakhile.harvey@gmail.com>Skype:
sakhile.harvey <http://sakhile.harvey>*

**Comment by smasoka** [comment]
The bug can be closed by #73
